public/plugin.ts

```diff
@@ -61,6 +71,66 @@ export class notificationsDashboardsPlugin
       });
     }
 
+    if (core.chrome.navGroup.getNavGroupEnabled()) {
+      // register applications with category and use case information
+      core.chrome.navGroup.addNavLinksToGroup(DEFAULT_NAV_GROUPS.settingsAndSetup, [
+        {
+          id: PLUGIN_NAME,
+          showInAllNavGroup: true
+        }
+      ])
+
+      // channels route
+      core.application.register({
+        id: `channels`,
+        title: 'Channels',
+        order: 9070,
+        workspaceAvailability: WorkspaceAvailability.outsideWorkspace,
+        mount: async (params: AppMountParameters) => {
+          return mountWrapper(params, ROUTES.CHANNELS);
+        },
+      });
+
+      core.application.register({
+        id: `email_senders`,
+        title: 'Email senders',
+        order: 9080,
+        workspaceAvailability: WorkspaceAvailability.outsideWorkspace,
+        mount: async (params: AppMountParameters) => {
+          return mountWrapper(params, ROUTES.EMAIL_SENDERS);
+        },
+      });
+
+      core.application.register({
+        id: `email_groups`,
+        title: 'Email recepient groups',
+        order: 9090,
+        workspaceAvailability: WorkspaceAvailability.outsideWorkspace,
```

Comment:
What is workspaceAvailability? and what are the other options used for? Could this be `workspace/no-workspace` distinction?